strings-csharp/FirstLetterToUpper/FirstLetterToUpperTests/FirstLetterToUpperUnitTests.cs

```diff
@@ -5,11 +5,11 @@ namespace FirstLetterToUpperTests
     [TestClass]
     public class FirstLetterToUpperUnitTests
     {
+        FirstLetterToUpperMethods upperCase = new FirstLetterToUpperMethods();
```

Comment:
Should be `var upperCase`. I've changed in the article 😃

Reply:
Hey @PhilBroderick, it didn't work because it's a global variable that's why I had to use FirstLetterToUpperMethods upperCase  instead of var

Reply:
Please advise